### PR TITLE
Add use cases

### PR DESCRIPTION
**Pull request by weizman**
closes: #4 

Add use cases based on those details on #4 

* maybe add some more links where possible? For example to the discussed projects/solutions/companies/blog-posts etc
* make sure both Shopify and Akamai approve of their part in this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,7 +215,85 @@ newFetchInstance(`https://${server}/${path}/?payload=` + payload)
...
 ## Use Cases
 
-> *Adding use cases is a WIP - if you find this proposal useful (or just want to explore some potential use cases), help by visiting [#4](https://github.com/weizman/Realms-Initialization-Control/issues/4) and share your use case to push this proposal forward!*
+Here are some use cases introduced by the community which led to composing this proposal.
+
+> _Please note that on our attempt to list use cases, we link them to real life examples by providing names of companies and/or products - none of them has actively reached out and provided their feedback on this proposal, but are merely given as use cases examples (unless clearly stated so at the top of each use case)._
+
+### Safe composability (sandboxing / confinement)
+
+> _This use case is approved by [Shopify](https://github.com/shopify) and [MetaMask](https://github.com/metamask)_
```
Comment (yoavweiss) [inline README.md:224]: I would go with something significantly more succinct and generic:

"""
The ability to safely embed untrusted code in a safe way is important for composability. Platforms can use it to allow their users as well as third party providers to enhance their provided functionality, and provide value to end users.

While the web platform allows such safe embedding through cross-origin iframes or workers, there are cases where same-origin untrusted-code embedding is required. E.g., in cases where the untrusted code may not be custom tailored to a sandboxed environment. 
In such cases, in order to guarantee the end user's security, platforms will constrain the capabilities that are available to the untrusted code by overriding native prototypes.

At the same time, it's hard to ensure that the overriding scripts are first to run in the embedded contexts, and determined attackers can use that to escape the sandbox under certain conditions. See [the same origin concern](https://weizmangal.com/2023/09/28/the-same-origin-concern/) for more details on that.

This proposal aims to solve this, by enabling the top-level document to guarantee running an initial script in every new same-origin realm, and prevent races that allow for sandbox escapes.
"""

Comment (weizman) [reply README.md]: cff6cb7

**Review by yoavweiss — APPROVED** [review]
LGTM with a slightly tighter/clearer phrasing. Your call..

File: README.md [hunk 2]
```diff
@@ -215,7 +215,50 @@ newFetchInstance(`https://${server}/${path}/?payload=` + payload)
...
+While the web platform allows such safe embedding through cross-origin iframes or workers, there are cases where same-origin untrusted-code embedding is required. E.g., in cases where the untrusted code may not be custom tailored to a sandboxed environment.
+In such cases, in order to guarantee the end user's security, platforms will constrain the capabilities that are available to the untrusted code by overriding native prototypes.
+
+At the same time, it's hard to ensure that the overriding scripts are first to run in the embedded contexts, and determined attackers can use that to escape the sandbox under certain conditions. See the same origin concern for more details on that.
+
+This proposal aims to solve this, by enabling the top-level document to guarantee running an initial script in every new same-origin realm, and prevent races that allow for sandbox escapes.
+
+### Application Monitoring (security / errors / performance / ux)
+
+The ability to provide successful monitoring services for web applications heavily relies on the stage in which they are included within the different JavaScript realms of the app they're expected to monitor.
```
[the remark this answers is not in the document]
Comment (weizman) [reply README.md]: how's this? 1726e1e7e14feb26c756edd6d4d1f7e4c0bf145c

[the remark this answers is not in the document]
Comment (yoavweiss) [reply README.md]: Looks great!

**Pull request merged**